weblate/vcs/git.py

```diff
@@ -669,10 +668,7 @@ def create_fork(self):
             data={},
         )
```

Comment:
Does this return name of existing fork in case one already exists? If it does it would be great to mention this in a comment, otherwise this case needs to be handled.

Reply:
That's a good point. I will test this.

Reply:
In Github API it does return the name (and all details) of the existing fork. I will add a comment to reflect the same.